app/models/manageiq/providers/ansible_tower/inventory/parser/automation_manager.rb

```diff
@@ -43,10 +51,51 @@ def configuration_script_sources
       i.playbooks.each do |playbook_name|
         # FIXME: its not really nice how I have to build a manager_ref / uuid here
         p = target.playbooks.find_or_build("#{i.id}__#{playbook_name}")
-        # FIXME: how about just adding `o` - configuration_script_source here?
-        p[:configuration_script_source] = target.configuration_script_sources.lazy_find(i.id.to_s)
+        p[:configuration_script_source] = o
         p[:name] = playbook_name
       end
     end
   end
+
+  def credentials
+    collector.credentials.each do |i|
+      o = target.configuration_script_authentications.find_or_build(i.id.to_s)
```

Comment:
I believe it's `AuthenticationConfigurationScriptBases` unless this is something different.